### PR TITLE
Golang dep dependency management

### PR DESCRIPTION
**Pull request by aajdinov**
Switched to [goland/dep](https://golang.github.io/dep/) dependency management tool.

### CODE REVIEW

**Comment by nurfet-becirevic** [comment]
The dependency management works fine. Just remove these two lines from the build script.
https://github.com/StackPointCloud/oneandone-cloudserver-cli/blob/golang_dep_dependency_management/build/build.sh#L8-L9

**Comment by nurfet-becirevic** [comment]
The tool version needs to be updated.
https://github.com/StackPointCloud/oneandone-cloudserver-cli/blob/golang_dep_dependency_management/version#L1

File: sshkey.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func init() {
 							Usage: "Description of the SSH Key.",
```
Comment (nurfet-becirevic) [inline sshkey.go]: Could you add the short form for the SSH key name flag, `Name:  "name, n",` ?

**Comment by nurfet-becirevic** [comment]
We should remove ssh key from the table view because the key is too long. However wants to obtain the public keys should set `--json` flag.

image

**Comment by nurfet-becirevic** [comment]
Overall, looks good. Please make the requested changes. I am not sure which version should be set since the version 1.2.0 is already released [here](https://github.com/1and1/oneandone-cloudserver-cli/releases/tag/v1.2.0) without binaries. 
I have built all binaries and tested windows installer.

**Comment by edevenport** [comment]
I deleted the v1.2.0 tagged release since it was not ready. I will wait until this PR is merged and we have assets to upload along with the v1.2.0 release.

**Comment by aajdinov** [comment]
@nurfet-becirevic Thank you for the review. I've made the suggested changes. If it looks ok now, we can proceed to merge the PR.

**Pull request merged**